R/utils.r

```diff
@@ -124,7 +124,19 @@ first_upper <- function(x) {
 
 download <- function(path, url, ...) {
   request <- httr::GET(url, ...)
-  httr::stop_for_status(request)
+
+  if (httr::status_code(request) >= 400) {
+    # Parse the response if it is from GitHub, use stop_for_status otherwise.
+    tryCatch(github_response(request),
```

Comment:
Couldn't we put the `stop_for_status()` in the github response? I think that would make this cleaner.